attributes/default.rb

```diff
@@ -6,5 +6,9 @@
 
 default['mailcatcher']['bin'] = 'mailcatcher'
 default['mailcatcher']['version'] = '0.6.1'
+default['mailcatcher']['ruby_version'] = '2.1.7'
```

Comment:
The latest stable ruby version is `2.2.3` which is also available to install via rbenv. Can you change it to that one if there are no know issues with it?